src/Game.php

```diff
@@ -7,6 +7,10 @@
 use Monolog\Logger;
 
 use LotGD\Core\Models\Character;
+use LotGD\Core\Exceptions\ {
+    SceneNotFoundException,
+    ActionNotException
+};
```

Comment:
Technically PSR2 does not allow this aggregated use statement. PSR12 does but that's not accepted yet. I dont really care honestly :)